ChangeLog.md

```diff
@@ -1,5 +1,212 @@
 # Changelog for haskell-language-server
 
+## 1.2.0
+
+We finally release a new version of Haskell Language Server!
```

Comment:
This sentence threw me for a loop the first few times I read it. Maybe something like this instead?

```suggestion
We have finally released a new version of Haskell Language Server!
```

Reply:
or if you want something more active

```suggestion
We are finally releasing a new version of Haskell Language Server!
```